xcpmd/src/battery.c

```diff
@@ -572,7 +573,6 @@ int update_battery_status(unsigned int battery_index) {
         status.remaining_capacity = status.energy_now;
     }
 
-    closedir(battery_dir);
```

Comment:
@jandryuk This seems wrong, your fix should already have been merged for the ```closedir(battery_dir)``` error.  Gotta dig into this.

Reply:
Yep, went too far back with the rebase so it changed the hashes and included them in the PR.  Should be good now.